cmd/clusterctl/cmd/config_repositories.go

```diff
@@ -34,15 +35,55 @@ const (
 	RepositoriesOutputYaml = "yaml"
 	// RepositoriesOutputText is an option used to print the repository list in text format.
 	RepositoriesOutputText = "text"
+	// RepositoriesOutputName is an option used to print the repository list by name only.
+	RepositoriesOutputName = "name"
```

Comment:
I'm wondering if, instead of conflating requirements for `clusterctl completion` under `clusterctl config repositories`, we should instead add `clusterctl completion repositories` specifically for this scope

Reply:
We've already used `clusterctl completion <shell>`, so I could not add `repositories` subcommand for `clusterctl completion` :cry: